src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerManager.php

```diff
@@ -62,64 +64,48 @@ public function processResourceEvents(array $plugins, array $ids = []): void
      */
     protected function triggerEvents(EventResourceQueryContainerPluginInterface $plugin, array $ids = []): void
     {
-        if ($ids) {
```

Comment:
why did you remove this?

Reply:
we don't need it anymore. it's not enough for the event